features/other_a/profile_edit.feature

```diff
@@ -44,6 +44,22 @@ Scenario: Changing email address - entering an invalid email address
   Then I should see "Email does not seem to be a valid address"
     And 0 emails should be delivered
 
+Scenario: Changing email address - case-insensitive comparison
+
+  When I follow "Change Email"
+    And I fill in "New Email" with "foo@example.com"
+    And I fill in "Confirm New Email" with "FoO@example.com"
+    And I enter my password
```

Comment:
You can use `And I fill in "Password" with "password"` instead of adding a new step here.